source/scss/components/_messages.scss

```diff
@@ -0,0 +1,62 @@
+.fti-message {
+  color: $white;
+  padding: .875rem .9375rem;
+
+  &.info-message {
+    background-color: $cyan;
+  }
+
+  .message-container {
+    align-items: center;
+    display: flex;
+    min-height: 60px;
+
+    .icon-container {
+      align-items: center;
+      display: flex;
+      height: 24px;
+      justify-content: center;
+      width: 24px;
+
+      &.left-icon-container {
+        margin-right: 10px;
+      }
+
+      &.right-icon-container {
+        margin-left: 16px;
+      }
+    }
+
+    p {
+      font-size: 1rem;
+      line-height: 1.25;
+      margin: 0;
+      width: 100%;
+
+      a {
+        color: $white;
+        text-decoration: underline;
+      }
+    }
+  }
+}
+
+@include media-breakpoint-up("lg") {
+  .fti-message {
+    padding: 1.25rem 3.75rem;
+  
+    .message-container {
+      min-height: 20px;
```

Comment:
I think it is good if the content defines the height, so maybe you know of a reason, why there is a min-height. Perhaps there is a way of creating distance from content to element border with a padding.